CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
-## 1.0.5 - TBD
+## 1.1.0 - TBD
```

Comment:
Um... this was correct. I think this was likely the result of a bad merge when you re-targeted. I'll fix this file on merge.